manifest-json-schema.json

```diff
@@ -29,26 +41,18 @@
       "description": "The path where the build logs should be written to.",
       "default": "/var/log/otp-build.log"
     },
-    "graphObjUrl": {
+    "graphObjUri": {
       "type": "string",
-      "description": "A url where the Graph.obj should be downloaded from for server-only runs. If `buildGraph` is set to false and `runServer` is set to true, this value must be defined. If `uploadGraph` is set to true, this value must be an s3 url that can be uploaded to."
-    },
-    "gtfsDownloads": {
-      "type": "array",
-      "description": "An array of URLs that should be downloaded to the base folder (see description in `baseFolder` for what this folder will be). Each of these files will be prepended with the string `gtfs` in order for OTP 2.x to properly recognize the file as a valid GTFS file.",
-      "items": {
-        "type": "string",
-        "description": "Either a URL or AWS S3 URL."
-      }
+      "description": "Either an HTTP(S) URL or AWS S3 URI where the Graph.obj should be downloaded from for server-only runs. If `buildGraph` is set to false and `runServer` is set to true, this value must be defined. If `uploadGraph` is set to true, this value must be an AWS S3 URI that can be uploaded to."
     },
     "jarFile": {
       "type": "string",
       "description": "The full path to the OTP jar file.",
       "default": "/opt/otp-1.4.0-shaded.jar"
     },
-    "jarUrl": {
+    "jarUri": {
```

Comment:
Out of curiosity, does the schema validator support rejecting config properties not listed in this file?

Reply:
It does not fail (given everything else is OK) if there are unsupported config properties in the file.